example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -1087,14 +1089,16 @@ describe('The S3 Ingest Granules workflow', () => {
         // Check that the granule was removed
         await waitForConceptExistsOutcome(cmrLink, false);
         const doesExist = await conceptExists(cmrLink);
-        expect(doesExist).toEqual(false);
+        // because of stubbed conceptExists due to https://github.com/nasa/cumulus/pull/3544/
```

Comment:
```suggestion
        // because of stubbed conceptExists due to https://github.com/nasa/cumulus/pull/3544/. This should be reverted to 'false' when the stub is removed.
```